<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainW</class>
 <widget class="QWidget" name="mainW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>475</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FlashForward</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #313333;
color: #FFFFFF;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,1,0,0,0,0">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="ffmpegHBL">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="ffmpegL">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #a6a6a6;</string>
       </property>
       <property name="text">
        <string>FFmpeg Dir:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ffmpegLE">
       <property name="styleSheet">
        <string notr="true">background-color: #202222; color: #FFFFFF; selection-background-color: #E4EAA0; selection-color: #313333</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ffmpegTB">
       <property name="styleSheet">
        <string notr="true">background-color: #313333; color: #f8f8f8;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="modeHBL">
     <item>
      <widget class="QPushButton" name="renderPBT">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #E4EAA0; color: #313333</string>
       </property>
       <property name="text">
        <string>Render</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playPBT">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #313333; color: #f8f8f8;</string>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="probePBT">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #313333; color: #f8f8f8;</string>
       </property>
       <property name="text">
        <string>Probe</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="modeS">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="optionF">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>194</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="optionVBL">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="inputHBL">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="inputL">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #a6a6a6;</string>
            </property>
            <property name="text">
             <string>Input:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputLE">
            <property name="styleSheet">
             <string notr="true">background-color: #202222; color: #FFFFFF; selection-background-color: #E4EAA0; selection-color: #313333</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="inputTB">
            <property name="styleSheet">
             <string notr="true">background-color: #313333; color: #f8f8f8;</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="outputHBL">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="outputL">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #a6a6a6;</string>
            </property>
            <property name="text">
             <string>Output:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="outputLE">
            <property name="styleSheet">
             <string notr="true">background-color: #202222; color: #FFFFFF; selection-background-color: #E4EAA0; selection-color: #313333</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="outputTB">
            <property name="styleSheet">
             <string notr="true">background-color: #313333; color: #f8f8f8;</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="paramVBL">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="codecHBL" stretch="0,1">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="codecL">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>0</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #a6a6a6;</string>
              </property>
              <property name="text">
               <string>Codec:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="codecCB">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #202222; color: #FFFFFF; selection-background-color:  #E4EAA0; selection-color: #202222</string>
              </property>
              <item>
               <property name="text">
                <string>apply_trc iec61966_2_1</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="resolutionHBL" stretch="0,1">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="resolutionL">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>0</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #a6a6a6;</string>
              </property>
              <property name="text">
               <string>Resolution:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="resolutionCB">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #202222; color: #FFFFFF; selection-background-color:  #E4EAA0; selection-color: #202222</string>
              </property>
              <item>
               <property name="text">
                <string>1920x1080 Progressive</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="frameRateHBL" stretch="0,1">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="frameRateL">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>0</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #a6a6a6;</string>
              </property>
              <property name="text">
               <string>Frame Rate:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="frameRateCB">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #202222; color: #FFFFFF; selection-background-color:  #E4EAA0; selection-color: #202222</string>
              </property>
              <item>
               <property name="text">
                <string>24</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="colorspaceHBL" stretch="0,1">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="colorspaceL">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>0</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #a6a6a6;</string>
              </property>
              <property name="text">
               <string>Colorspace:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="colorspaceCB">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #202222; color: #FFFFFF; selection-background-color:  #E4EAA0; selection-color: #202222</string>
              </property>
              <item>
               <property name="text">
                <string>Linear to sRGB</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="miscHBL" stretch="0,0,0,0">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QCheckBox" name="slateCB">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #a6a6a6;</string>
              </property>
              <property name="text">
               <string>Slate</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="burninCB">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #a6a6a6;</string>
              </property>
              <property name="text">
               <string>Burn-in</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="noteL">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>0</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #a6a6a6;</string>
              </property>
              <property name="text">
               <string>Note:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="noteLE">
              <property name="styleSheet">
               <string notr="true">background-color: #202222; color: #FFFFFF; selection-background-color: #E4EAA0; selection-color: #313333</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="optionS">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="queueVBL" stretch="0,1">
     <item>
      <layout class="QHBoxLayout" name="queueHBL">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="queueL">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>20</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #a6a6a6;</string>
         </property>
         <property name="text">
          <string>Queue:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="queueS">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="queueRemovePB">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #313333; color: #f8f8f8;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="queueAddPB">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #313333; color: #f8f8f8;</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="queueLW">
       <property name="styleSheet">
        <string notr="true">background-color: #202222; color: #FFFFFF; selection-background-color: #E4EAA0; selection-color: #313333;

/*
QListWidget::item
{
    background-color: #202222;
    color: #FFFFFF;
    selection-background-color: #313333;
    selection-color: #E4EAA0;
};

QListWidget::item:selected
{
    background-color: #202222;
    color: #FFFFFF;
    selection-background-color: #313333;
    selection-color: #E4EAA0;
};
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Fixed</enum>
       </property>
       <property name="viewMode">
        <enum>QListView::ListMode</enum>
       </property>
       <item>
        <property name="text">
         <string>test1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>test2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>test3</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="buttonsHL">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>3</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsHBL">
     <item>
      <widget class="QPushButton" name="runPB">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #E4EAA0; color: #313333</string>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="copyrightHL">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>3</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="copyrightHBL" stretch="0,0,0">
     <item>
      <spacer name="copyrightS_L">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="copyrightL">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #a6a6a6;</string>
       </property>
       <property name="text">
        <string>© 2024 Q for Quesadilla. All Rights Reserved.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="copyrightS_R">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
